extensions/functions_arithmetic.yaml

```diff
@@ -1369,6 +1369,109 @@ aggregate_functions:
         ordered: true
         return: LIST?<any>
 
+  - name: "bit_or"
+    description: >
+      Calculates the bitwise OR of all values in set of values.
+    impls:
+      - args:
+          - name: x
+            value: i8
+        nullability: DECLARED_OUTPUT
+        return: i8
+      - args:
+          - name: x
+            value: i16
+        nullability: DECLARED_OUTPUT
+        return: i16
+      - args:
+          - name: x
+            value: i32
+        nullability: DECLARED_OUTPUT
+        return: i32
+      - args:
+          - name: x
+            value: i64
+        nullability: DECLARED_OUTPUT
+        return: i64
+      - args:
+          - name: x
+            value: fp32
+        nullability: DECLARED_OUTPUT
+        return: fp32
+      - args:
+          - name: x
+            value: fp64
+        nullability: DECLARED_OUTPUT
+        return: fp64
```

Comment:
Bitwise operations on floats make no sense and should be removed.

Reply:
Ah... thanks for catching this one.